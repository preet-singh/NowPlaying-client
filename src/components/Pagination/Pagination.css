.pagination {
    display: flex;
    flex-direction: row;
    justify-content: center;
}

.pagination .black-button {
    margin: 5px;
}