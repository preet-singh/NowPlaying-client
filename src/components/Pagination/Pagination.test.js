import React from 'react';
import ReactDOM from 'react-dom';
import Pagination from './Pagination';
import { MemoryRouter } from 'react-router-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json'

describe('Pagination testing', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <MemoryRouter>
            <Pagination />
        </MemoryRouter>, 
    div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
  it('paginates the posts', () => {
    const wrapper = shallow(<Pagination />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
  
});