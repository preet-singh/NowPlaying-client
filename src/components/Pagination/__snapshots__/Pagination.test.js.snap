// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination testing paginates the posts 1`] = `
<nav>
  <ul
    className="pagination"
  />
</nav>
`;
